Block/NodeType/AbstractNode.php

```diff
@@ -125,4 +130,16 @@ public function isViewAllLinkAllowed()
     {
         return $this->viewAllLink;
     }
+
+    public function _toHtml()
```

Comment:
I think it should be `protected` because you override the parent class method.

Reply:
updated to protected